t/plugin/error-log-logger-kafka.t

```diff
@@ -87,7 +87,8 @@ done
                             "host": "127.0.0.1",
                             "port": 9092
                         }],
-                        "kafka_topic": "test2"
+                        "kafka_topic": "test2",
+                        "meta_refresh_interval": 1
```

Comment:
Can we add a mock test to ensure this config is passed to broker_config?

Reply:
@spacewander Can we check by the following steps:

1. log the `broker_config` info in `send_to_kafka` function
2. check if the error log include `"refresh_interval":1000` or not?

Reply:
We can verify it without adding any code to the plugin.

https://github.com/apache/apisix/blob/0bc65ea9acd726f79f80ae0abd8f50b7eb172e3d/t/plugin/sls-logger.t#L373
Like this, we can inject some code while creating the producer.

Reply:
ok, i see. But func `create_producer` is a local func, how can we inject it by `extra_init_by_lua`?

Reply:
We can inject https://github.com/apache/apisix/blob/0bc65ea9acd726f79f80ae0abd8f50b7eb172e3d/apisix/plugins/error-log-logger.lua#L351

Reply:
@spacewander updated, please take a look, thanks.